AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/efragment/MyListFragmentTest.java

```diff
@@ -123,6 +123,28 @@ public void ignoredWhenDetachedWorksForIgnoredMethod() {
 	}
 
 	@Test
+	public void ignoredWhenViewDestroyedForIgnoredMethod() {
+		assertFalse(myListFragment.didExecute);
+		myListFragment.onDestroyView();
+		myListFragment.ignoreWhenViewDestroyed();
+		assertFalse(myListFragment.didExecute);
+	}
+
+	@Test
+	public void notIgnoredBeforeDetached() {
+		assertFalse(myListFragment.didExecute);
+		myListFragment.ignored();
+		assertTrue(myListFragment.didExecute);
+	}
+
+	@Test
+	public void notIgnoredBeforeViewDestroyed() {
+		assertFalse(myListFragment.didExecute);
+		myListFragment.ignoreWhenViewDestroyed();
+		assertTrue(myListFragment.didExecute);
+	}
+
```

Comment:
No worry. 

I added more test case and squash. Check this method. https://github.com/excilys/androidannotations/pull/1664/files#diff-5474a5decd9ffab9908ae97c83444b47R143

In case you don't want me to commit & squash, let me know.